util/logs/log-k8s-requests.go

```diff
@@ -17,7 +17,7 @@ func (m k8sLogRoundTripper) RoundTrip(r *http.Request) (*http.Response, error) {
 	x, err := m.roundTripper.RoundTrip(r)
 	if x != nil {
 		verb, kind := k8s.ParseRequest(r)
-		log.Debugf("%s %s %d", verb, kind, x.StatusCode)
+		log.Warnf("%s %s %d", verb, kind, x.StatusCode)
```

Comment:
What's the motivation?

Reply:
it should be kept at debug, this log is very expensive!